### PR TITLE
OSDOCS-11142: 4.15.20 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.15

Issue:
[OSDOCS-11142](https://issues.redhat.com/browse/OSDOCS-11142)

Link to docs preview:
https://78383--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes.html#ocp-4-15-20

QE review:
N/A

Additional information:
Errata URLs may return 404 until go-live (7/02)

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2733,6 +2733,62 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, AWS HyperShift clusters leveraged their VPC's primary CIDR range to generate security group rules on the data plane. As a consequence, installing AWS HyperShift clusters into an AWS VPC with multiple CIDR ranges caused the generated security group rules to be insufficient. With this release, security group rules are generated based on the provided Machine CIDR range instead to resolve this issue. (link:https://issues.redhat.com/browse/OCPBUGS-35714[*OCPBUGS-35714])
+
+* Previously, in User Provisioned Infrastructure (UPI) or clusters that were upgraded from older versions, `failureDomains` may be missing in Infrastructure objects which caused certain checks to fail. With this release, a fallback `failureDomains` is synthesized from `cloudConfig` if none are available in `infrastructures.config.openshift.io`. (link:https://issues.redhat.com/browse/OCPBUGS-35446[*OCPBUGS-35446*])
```
Comment (sabrinajess) [inline release_notes/ocp-4-15-release-notes.adoc:2777]: Change "may" to "might" per ISG.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -2733,6 +2733,62 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, in User Provisioned Infrastructure (UPI) or clusters that were upgraded from older versions, `failureDomains` may be missing in Infrastructure objects which caused certain checks to fail. With this release, a fallback `failureDomains` is synthesized from `cloudConfig` if none are available in `infrastructures.config.openshift.io`. (link:https://issues.redhat.com/browse/OCPBUGS-35732[*OCPBUGS-35732*])
```
Comment (sabrinajess) [inline release_notes/ocp-4-15-release-notes.adoc:2779]: Change "may" to "might" per ISG.

**Review by sabrinajess — COMMENTED** [review]
A couple nits but otherwise LGTM!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (obrown1205) [reply release_notes/ocp-4-15-release-notes.adoc]: good catch! ty!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 2 above)
Comment (obrown1205) [reply release_notes/ocp-4-15-release-notes.adoc:2777]: good catch! ty!

[the remark this answers is not in the document]
Comment (obrown1205) [reply release_notes/ocp-4-15-release-notes.adoc:2777]: good catch! ty!

**Pull request merged**